style.css

```diff
@@ -0,0 +1,94 @@
+.body {
+  max-width: 1920px;
+  min-width: 375px;
+  height: 100vh;
+  margin: 0 auto;
+  background: url('pic/iswanto-arif-448003-unsplash.png');
+  background-size: cover;
+}
+.header {
+  width: 100%;
+  padding: 89px 114px 0 200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+}
+.nav {
+  display: flex;
+  justify-content: center;
+}
+.nav__link:not(:last-child) {
+  margin: 0 34px 0 0;
+}
+.nav__link{
+  color: #fff;
+  text-decoration: none;
+  font-size: 22px;
+  font-family: 'Museo';
+}
+  .container{
+    display: flex;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)